spec/features/users_index_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'rails_helper'
+RSpec.describe 'root page features' do
+  before(:each) do
+    @users = [
+      @user1 = User.create(
+        name: 'Bal',
+        photo: 'https://img.freepik.com/free-vector/mysterious-mafia-man-smoking-cigarette_52683-34828.jpg?w=1480&t=st=1674661161~exp=1674661761~hmac=e529680000ea966150e5ea3b38241a0d8c9582faf23dac8af61ce1785cf27838',
+        bio: 'Fullstack Web developer from Uganda',
+        posts_counter: 0
+      ),
+      @user2 = User.create(
+        name: 'John',
+        photo: 'https://img.freepik.com/free-vector/mysterious-mafia-man-smoking-cigarette_52683-34828.jpg?w=1480&t=st=1674661161~exp=1674661761~hmac=e529680000ea966150e5ea3b38241a0d8c9582faf23dac8af61ce1785cf27838',
+        bio: 'Teacher from Poland.',
+        posts_counter: 0
+      ),
+      @user3 = User.create(
+        name: 'Balix',
+        photo: 'https://img.freepik.com/free-vector/mysterious-mafia-man-smoking-cigarette_52683-34828.jpg?w=1480&t=st=1674661161~exp=1674661761~hmac=e529680000ea966150e5ea3b38241a0d8c9582faf23dac8af61ce1785cf27838',
+        bio: 'Teacher from Mexico.',
+        posts_counter: 0
+      )
+    ]
+    visit('/')
+  end
+  describe 'users#index' do
+    it 'displays all the different usernames' do
+      @users.each do |user|
+        expect(page).to have_content(user.name)
+      end
+    end
+    it 'display the profile picture for each user' do
+      @users.each do |user|
+        expect(page).to have_css("img[src='#{user.photo}']")
+      end
+    end
+    it 'displays number of posts for each user' do
+      @users.each do |user|
+        expect(page).to have_content("Number of posts: #{user.posts_counter}")
```

Comment:
* Kindly add a test for this missing case `When I click on a user, I am redirected to that user's show page.` as required in the [requirements](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/intergration_spec_and_n_1_project.md#:~:text=When%20I%20click%20on%20a%20user%2C%20I%20am%20redirected%20to%20that%20user%27s%20show%20page.)

Reply:
Thank you @Davidosky007 for the constructive feedback.Let me fix the review comment.